modules/op-installing-pipelines-operator-using-the-cli.adoc

```diff
@@ -3,14 +3,13 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="op-installing-pipelines-operator-using-the-cli_{context}"]
-= Installing the {pipelines-shortname} Operator using the CLI
+= Installing the {pipelines-shortname} Operator by using the CLI
 
-You can install {pipelines-title} Operator from the OperatorHub using the CLI.
+You can install {pipelines-title} Operator from the OperatorHub by using the CLI.
```

Comment:
```suggestion
You can install {pipelines-title} Operator from OperatorHub by using the command-line interface (CLI).
```

- Remove `the`: I see more search results in repo for "OperatorHub" than "the OperatorHub".
- Expand acronym (CLI) for its first occurrence in the module.